dev/drivers/scripts/plots/global_ens/jevs_global_ens_atmos_naefs_precip_spatial_plots.sh

```diff
@@ -0,0 +1,44 @@
+#PBS -N jevs_global_ens_atmos_naefs_precip_spatial_plots
+#PBS -j oe 
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:30:00
```

Comment:
The ecf script has `#PBS -l walltime=00:25:00`. Can you update one of them so they match?

Reply:
Just did.